lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -15,17 +15,28 @@
     db_truncate('dosomething_rogue_failed_task_log')->execute();
 
     foreach ($task_log as $task) {
-      $values = [
-        'nid' => $task->campaign_id,
-        'campaign_run_id' => $task->campaign_run_id,
-        'quantity' => $task->quantity,
-        'why_participated' => $task->why_participated,
-        'file' => $task->file,
-        'caption' => $task->caption,
-      ];
+      if ($task->type === 'reportback') {
+        $values = [
+          'nid' => $task->campaign_id,
+          'campaign_run_id' => $task->campaign_run_id,
+          'quantity' => $task->quantity,
+          'why_participated' => $task->why_participated,
+          'file' => $task->file,
+          'caption' => $task->caption,
+        ];
 
-      $user = user_load($task->drupal_id);
+        $user = user_load($task->drupal_id);
```

Comment:
nono totally - just saw that too and saw it looked weird! I think it is ok though because I wrapped that in an `if` statement so now should be indented to line up with the `$values` array above?